chat/run.sh

```diff
@@ -15,4 +15,10 @@
 # limitations under the License.
 #
 # SPDX-License-Identifier: Apache-2.0
-python -m llama_cpp.server --model ${MODEL_PATH} --host ${HOST:=0.0.0.0} --port ${PORT:=8000} --n_gpu_layers 0
+if [ ${MODEL_PATH} ]; then
+    python -m llama_cpp.server --model ${MODEL_PATH} --host ${HOST:=0.0.0.0} --port ${PORT:=8001} --n_gpu_layers ${GPU_LAYERS:=0} --clip_model_path ${CLIP_MODEL_PATH:=None} --chat_format ${CHAT_FORMAT:="llama-2"}
```

Comment:
```suggestion
    python -m llama_cpp.server --model ${MODEL_PATH} --host ${HOST:=0.0.0.0} --port ${PORT:=8001} --n_gpu_layers ${GPU_LAYERS:=0} --clip_model_path ${MODEL_CLIP_MODEL_PATH:=None} --chat_format ${MODEL_CHAT_FORMAT:="llama-2"}
```

unsure if MODEL_CLIP_MODEL_PATH or MODEL_CLIP_PATH

Reply:
GPU_LAYERS and CLIP_MODEL_PATH should probably be not changed, as if used it will be configured by AI-lab, through the creation form, as it would not make much sense to have preconfigured values for them.

GPU_LAYERS is hardware specific, it is the number of layers of the models that the GPU will take. E.g. I have 33 layers on llama2, and I am sending 15 to the GPU since I have a GPU with not a lot of ram. 

CLIP_MODEL_PATH from my understanding would be a path that we would have to provide, therefore it could be anywhere we we want, as we would have to mount the volume in the expected place

Reply:
CLIP_MODEL_PATH is going to be handled differently when AI Lab implements multi file models as we had an agreement that it will be given in the MODEL_PATH variable